### ISSUE TITLE
Block.setstyle throws an error when block has no style name

### ISSUE THREAD

**Issue by alschmiedt** [google/blockly#3343]
**Describe the bug**
block.setStyle throws an error when there is a block with no style name and the theme is changed.

**To Reproduce**
1. Go to the playground
2. Go to test blocks and drag out the top block 
3. Try changing the theme
Observe: ALL THE ERRORS

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Stack Traces**

```
Uncaught Error: Invalid style name: null
    at Blockly.BlockSvg.Blockly.Block.setStyle (block.js:1013)
    at Blockly.WorkspaceSvg.Blockly.Workspace.updateBlockStyles_ (workspace.js:255)
    at Blockly.WorkspaceSvg.Blockly.Workspace.refreshTheme (workspace.js:235)
    at Blockly.WorkspaceSvg.refreshTheme (workspace_svg.js:2567)
    at Blockly.ThemeManager.setTheme (theme_manager.js:94)
    at Blockly.WorkspaceSvg.Blockly.Workspace.setTheme (workspace.js:226)
    at changeTheme (playground.html?dir=ltr&toolbox=test-blocks&side=start&renderer=geras:334)
    at HTMLSelectElement.onchange (playground.html?dir=ltr&toolbox=test-blocks&side=start&renderer=geras:591)
Blockly.Block.setStyle @ block.js:1013
Blockly.Workspace.updateBlockStyles_ @ workspace.js:255
Blockly.Workspace.refreshTheme @ workspace.js:235
Blockly.WorkspaceSvg.refreshTheme @ workspace_svg.js:2567
Blockly.ThemeManager.setTheme @ theme_manager.js:94
Blockly.Workspace.setTheme @ workspace.js:226
changeTheme @ playground.html?dir=ltr&toolbox=test-blocks&side=start&renderer=geras:334
onchange @ playground.html?dir=ltr&toolbox=test-blocks&side=start&renderer=geras:591```

**Additional context**

<!-- Add any other context about the problem here. -->

**Comment by samelhusseini** [comment]
I think I deal with this here https://github.com/google/blockly/pull/3334/files#diff-84b5fa00c539dcb35c0fc1e659c2a9d1R249

**Comment by alschmiedt** [comment]
Fixed in #3334
